dask/array/tests/test_array_core.py

```diff
@@ -2335,6 +2334,23 @@ def test_vindex_merge():
                                                           [10, 20, 30]])).all()
 
 
+def test_vindex_identity():
+    rng = da.random.RandomState(42)
+    a, b = 10, 20
+
+    x = rng.random(a, chunks=a // 2)
+    assert x is x.vindex[:]
```

Comment:
Can we also test `x.vindex[:10]` here for completeness?

Reply:
Good catch.

Reply:
Good catch.